src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -4890,7 +4890,7 @@ private static MethodHandle identityOrVoid(Class<?> type) {
      * @param type the type of the desired method handle
      * @return a constant method handle of the given type, which returns a default value of the given return type
      * @throws NullPointerException if the argument is null
-     * @see MethodHandles#primitiveZero
+     * @see <code>primitiveZero(Wrapper)</code>
```

Comment:
Just `#zero(Class)` should be sufficient. This is probably changed by intellij during my original work to consolidate zero and constant.

Reply:
From <https://github.com/openjdk/jdk/pull/23706#discussion_r2094138280>:
```suggestion
     * @see MethodHandles#zero
```